### ISSUE TITLE
Lack of accessibility labels makes it hard to reply to comments

### ISSUE THREAD

**Issue by maxme** [wordpress-mobile/WordPress-iOS#6894]
User commented:

> Some of your buttons are not accessible to voice over so it makes it very hard to reply to comments.
